src/shell/kill.c

```diff
@@ -42,21 +42,11 @@ static void kill_cb (flux_t *h, flux_msg_handler_t *mh,
                      const flux_msg_t *msg, void *arg)
 {
     flux_shell_t *shell = arg;
-    uint32_t userid;
-    uint32_t rolemask;
     int signum;
-
-    if (flux_msg_get_userid (msg, &userid) < 0
-        || flux_msg_get_rolemask (msg, &rolemask) < 0) {
-        log_err ("kill: unable to read msg userid or rolemask");
-        return;
-    }
-    if (!(rolemask & FLUX_ROLE_OWNER) && userid != getuid ()) {
-        log_msg ("kill: invalid userid %u", userid);
-        return;
-    }
     if (flux_msg_unpack (msg, "{s:i}", "signum", &signum) < 0) {
-        log_msg ("kill: invalid kill event msg payload");
+        const char *p;
```

Comment:
I assume this was here to debug a problem, but it's perhaps overkill now?

Reply:
Yes, sorry that crept in!